core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -538,14 +538,17 @@ public String getDownloadUrl() {
 
     /**
      * Is this the legacy default update center site?
-     * @deprecated
-     *      Will be removed, currently returns always false.
-     * @since 2.343
+     * @since 1.357
      */
-    @Deprecated
```

Comment:
Reverting this hunk of #6116, since this functionality is useful and should not be deprecated.